### PR TITLE
docs: Cleanup version history file

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: docs: Cleanup version history file
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Review by alyssawilk — COMMENTED** [review]
Thanks for the clean up! Alas, looks like there's already merge conflicts.
Is there any valid use of single ticks around simple strings or should we fix-format this forward?
/wait

**Comment by phlax** [comment]
> Alas, looks like there's already merge conflicts.

ill resolve now...

> Is there any valid use of single ticks around simple strings or should we fix-format this forward?

not really no - it something we want to fix/check on an rst level tho - not with mad regexes i think

File: docs/root/version_history/current.rst [hunk 1]
```diff
@@ -92,14 +92,20 @@ New Features
...
 * http: a new field `is_optional` is added to `extensions.filters.network.http_connection_manager.v3.HttpFilter`. When
   value is `true`, the unsupported http filter will be ignored by envoy. This is also same with unsupported http filter
+* dns resolver: added ``DnsResolverOptions`` protobuf message to reconcile all of the DNS lookup option flags. By setting the configuration option :ref:`use_tcp_for_dns_lookups <envoy_v3_api_field_config.core.v3.DnsResolverOptions.use_tcp_for_dns_lookups>` as true we can make the underlying dns resolver library to make only TCP queries to the DNS servers and by setting the configuration option :ref:`no_default_search_domain <envoy_v3_api_field_config.core.v3.DnsResolverOptions.no_default_search_domain>` as true the DNS resolver library will not use the default search domains.
```
Comment (alyssawilk) [inline docs/root/version_history/current.rst:95]: do you know what's up here? the diffs don't look clean here unless you actually added content?

Comment (phlax) [reply docs/root/version_history/current.rst:89]: not sure what happened - im gonna blame git - either way, should be fixed now

**Review by alyssawilk — APPROVED** [review]
would love to see this land with a fix format change but I'm not going to block on it :-P

**Comment by phlax** [comment]
> would love to see this land with a fix format change but I'm not going to block on it :-P

its too complex for the scope of this pr

i would also love for us to have a more general solution to this problem - there are a couple of blockers to that:

- it needs to run against generated rst - i not just against this single file
- there is not an easy way to enforce a literal over emphasized text (altho spellcheck does this in part)
- there is not an easy way to check for ` ` ` over ` `` ` - this could be added, but it needs to work at the rst level to handle multi-lines etc

**Pull request merged**